tests/test_preprocess.py

```diff
@@ -172,14 +172,14 @@ def test_pca():
     preprocessor = Preprocessor()
     preprocessor.preprocess_adata_seurat_wo_pca(adata)
     adata = dyn.pp.pca(adata, n_pca_components=30)
-    assert adata.obsm["X"].shape[1] == 30
+    assert adata.obsm["X_pca"].shape[1] == 30
```

Comment:
you will need to update the pca_key in the pca function so that the default key is X_pca https://github.com/aristoteleo/dynamo-release/blob/master/dynamo/preprocessing/utils.py

def pca